src/vcpkg/binarycaching.cpp

```diff
@@ -2454,6 +2499,10 @@ void vcpkg::help_topic_binary_caching(const VcpkgPaths&)
                "**Experimental: will change or be removed without warning** Adds an AWS S3 source. "
                "Uses the aws CLI for uploads and downloads. Prefix should include s3:// scheme and be suffixed "
                "with a `/`.");
+    tbl.format("x-aws-config,<parameter>",
+               "**Experimental: will change or be removed without warning** Adds an AWS S3 source. "
+               "Adds an AWS configuration; currently supports only 'anonymous' parameter that is an equivalent to the "
+               "'----no-sign-request parameter of the AWS cli.");
```

Comment:
It's a bit lengthier but I would prefer `x-aws-config,no-sign-request` over `x-aws-config,anonymous` since it matches the parameter name and users don't have to learn a vcpkg specific mapping.

Reply:
Sounds reasonable. Done